modules/actor/actor-wfrp4e.js

```diff
@@ -3751,14 +3755,14 @@ export default class ActorWfrp4e extends Actor {
     if (this.isToken || token) {
       return {
         token: token?.id || this.token.id,
-        scene: token?.parent.id || this.token.parent.id
+        scene: token?.scene?.id || token?.parent?.id || this?.token?.scene?.id || this?.token?.parent?.id
       }
     }
     else {
       return {
         actor: this.id,
         token: token?.id,
-        scene: token?.parent.id
```

Comment:
What was this change for?

Reply:
i will have to re-test this, but when i did those changes, i had some cases, where this code was causing errors - sometimes token had parent - and sometimes scene. and sometimes there was no token for given actor on given scene.